src/strategies/planet-finance-v2/index.ts

```diff
@@ -4,11 +4,10 @@ import { Multicaller } from '../../utils';
 import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
 
 export const author = 'planet-finance';
```

Comment:
@defininja PR should be coming from `planet-finance`'s fork branch

Reply:
@ChaituVR - Hi 

I have changed author to defininja. Please check. Thanks.